t/base/protocols.t

```diff
@@ -1,20 +1,23 @@
 use strict;
 use warnings;
 use Test::More;
-plan tests => 6;
 
 use LWP::UserAgent;
+plan tests => 8;
 
 my $ua = LWP::UserAgent->new();
+isa_ok($ua, 'LWP::UserAgent', 'new: UserAgent instance');
```

Comment:
I'm also not sure how helpful these `isa_ok` checks are for `LWP::UserAgent`.  I can see it for a method that is supposed to return some kind of an object, but I'm not sure we're learning anything useful here.